pages/common/iperf.md

```diff
@@ -14,3 +14,15 @@
 - Run on client with 5 parallel threads:
 
 `iperf -c {{server_address}} -P {{5}}`
+
+- Run on client every 2 seconds:
+
+`iperf -c {{server_address}} -i {{2}}`
+
+- Run on server using UDP mode and set server port to listen on 5001:
```

Comment:
Wouldn't it make more sense to switch this example and the one below?

Reply:
yeah, @zdroid  I agree.

Reply:
Then the last example `- Run on server using UDP mode and set server port to listen on 5001:` should be moved up, no?

Reply:
@zdroid Take a look now.